multi-node/aws/cmd/kube-aws/command_render.go

```diff
@@ -32,16 +48,45 @@ func runCmdRender(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("Failed to read cluster config: %v", err)
 	}
 
-	// Generate default TLS assets.
-	assets, err := cluster.NewTLSAssets()
-	if err != nil {
-		return fmt.Errorf("Error generating default assets: %v", err)
-	}
-	if err := os.Mkdir("credentials", 0700); err != nil {
-		return err
-	}
-	if err := assets.WriteToDir("./credentials"); err != nil {
-		return fmt.Errorf("Error create assets: %v", err)
+	if renderOpts.generateCredentials {
+		fmt.Printf("Generating TLS credentials...\n")
+		var caKey *rsa.PrivateKey
```

Comment:
I don't think these should be pointers, or that NewTLSAssets should accept pointers.  It doesn't look like there's any reason for these to be nullable, and it looks like this problem goes all the way down to tlsutil, where eventually x509.CreateCertificate needs a pointer.  We should just take an address at that point rather than propagating the pointer type all the way up here.